web/ui/templates/_base.html

```diff
@@ -40,8 +40,9 @@
             <li><a href="{{ pathPrefix }}/alerts">Alerts</a></li>
             <li><a href="{{ pathPrefix }}/graph">Graph</a></li>
             <li><a href="{{ pathPrefix }}/status">Status</a></li>
+            <li><a href="{{ pathPrefix }}/config">Config</a></li>
```

Comment:
There's going to end up being a lot of these, I'd suggest linking from the status page rather than a tab each.